plugins/analysis-icu/src/main/java/org/elasticsearch/plugin/analysis/icu/ICUCollationKeyFilter.java

```diff
@@ -1,4 +1,12 @@
-package org.elasticsearch.index.analysis;
+/*
```

Comment:
This is a directly copy of lucene's old ICUCollationKeyFilter I think? In which case it should just keep the Apache 2 license header AIUI.